app/javascript/controllers/search_controller.js

```diff
@@ -0,0 +1,33 @@
+import { Controller } from "@hotwired/stimulus";
+
+// Connects to data-controller="search"
+export default class extends Controller {
+  static targets = ["input", "clearButton"];
+
+  // Trigger search on Enter key press
+  handleEnter(event) {
+    if (event.key === "Enter") {
```

Comment:
Why is there an empty if-statement here?

Reply:
Good observation, Max, and thank you for pointing it out. 

This line was not necessary, as I have recently found out that in HTML, a form gets "submitted" automatically when someone presses Enter after typing their query, and no additional event listeners are needed.

Accordingly, I have removed the `handleEnter` function and I have verified that the overall functionality of the search-bar has remained the same.